CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.17.2)
 
-project(VULKAN_TOOLS)
+project(VULKAN_TOOLS LANGUAGES CXX)
```

Comment:
By default for legacy reasons CMake enables both the C/C++ languages. This repo doesn't need that.

It makes things more complicated.

EX:

Needing to specify CMAKE_CXX_VISIBILITY_PRESET and CMAKE_C_VISIBILITY_PRESET and being more aware of the compiler flags being used.